### DISCUSSION TITLE
Is this project going to join the CNCF

### DISCUSSION THREAD

**Discussion by ewassef** [skupperproject/skupper#854] [General]
Curious about the long-term vision and if we should align with this project. We love it but are a bit curious about the long-term support and availability of the project. 
Will skupper join CNCF? or maybe some other foundation?

**Comment by grs** [comment]
I doubt anyone involved would object in skupper becoming a CNCF project. I hope that ultimately it will.

However it is one of those chicken-egg problems. I suspect the CNCF would want more evidence of wider backing before allowing it to enter, but others would prefer it to be part of CNCF or some other foundation before backing it.

For now I think the best we can do is to keep working on improving things, listening to users and potential users and trying to help them be successful with it.
